axum-macros/src/lib.rs

```diff
@@ -540,6 +540,29 @@ pub fn derive_from_request_parts(item: TokenStream) -> TokenStream {
 /// }
 /// ```
 ///
+/// # Limitations
+///
+/// This macro does not work for associated functions — functions defined in an impl block.
```

Comment:
I'm not sure everyone is gonna be familiar with what "associated functions" means. What do you think about clarifying it to say that it does work on methods that take `self`?